src/plone/restapi/services/actions/get.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.restapi.services import Service
+from zope.i18n import translate
+
+
+class ActionsGet(Service):
+
+    def translate(self, msg):
+        if self.request.get('translate'):
+            return translate(msg, context=self.request)
+        return msg
+
+    def reply(self):
+        context_state = api.content.get_view(name='plone_context_state',
+                                             context=self.context,
+                                             request=self.request)
+        data = {}
+        for category in ['object', 'object_buttons', 'user']:
```

Comment:
@sneridagh @robgietema do we want to restrict the API to just expose those categories?

Reply:
The implementation now returns all category and can be limited to certain categories with parameter.